configure.ac

```diff
@@ -74,8 +72,14 @@ AC_CHECK_LIB(dl, dlopen, [],
 dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  libfabric requires libpthread.]))
-AC_CHECK_LIB(rt, clock_gettime, [],
-    AC_MSG_ERROR([clock_gettime() not found.  libfabric requires librt.]))
+
+AC_SEARCH_LIBS([clock_gettime],[rt posix4],
+		[AC_DEFINE([HAVE_CLOCK_GETTIME], [1],
+			[Define to 1 if have clock_gettime.])],
+		[AS_IF([test -z $(expr "$host_os" : '.*\(darwin\)')],
+			[AC_MSG_ERROR([clock_gettime not found.])],
```

Comment:
Questions:

1. We've looked for clock_gettime in librt previously.  Now you're adding the ability to search through libposix4.  Is clock_gettime in libposix4 on Linux systems?
1. In the "else" condition, how about AC_CHECK_FUNCS for host_get_clock_service (and/or any of the other symbols you use in mach_clock_gettime)?